deps.qt/qt5.ps1

```diff
@@ -25,6 +25,11 @@ param(
 # path should also be kept short. This avoids issues with too long file paths in the compilation phase.
 
 function Setup {
+    if (${Target} -eq "arm64") {
```

Comment:
This should be implemented the same way as in the `zsh`-framework, with a local variable setting an allow-list and the dispatcher checking the the current target architecture is in that list.

Applies to all the similar changes below.